codes/extractive/netsum/netsum.py

```diff
@@ -43,18 +47,28 @@ def data_prep(text_path, summary_path, file_info, save_transformed_flag, transfo
             text = rf.read()
             text_sents = sent_tokenize(text)
             all_docs_words = word_tokenizer.tokenize(text)
-            text_word_counter = Counter(all_docs_words)
-            all_docs_bigram = list(nltk.bigrams(all_docs_words))
-            total_num_words = len(all_docs_words)
+            all_docs_words_cleaned = []
+            # add lemmatization here.
+            for w in all_docs_words:
+                all_docs_words_cleaned.append(lemmatizer.lemmatize(w,pos='v'))
+            text_word_counter = Counter(all_docs_words_cleaned)
+            all_docs_bigram = list(nltk.bigrams(all_docs_words_cleaned))
+            total_num_words = len(all_docs_words_cleaned)
     
         summary_file = os.path.join(summary_path,data_df["summary_path"][i])
         with open(summary_file, 'r', encoding="utf8") as rf:
             text = rf.read()
             summary_array = word_tokenizer.tokenize(text)
+            summary_array_cleaned = []
+            # add lemmatization here.
+            for w in summary_array:
+                summary_array_cleaned.append(lemmatizer.lemmatize(w,pos='v'))
         
         # focing the same shape.
         if len(text_sents) > MAX_INPUT_SEQ_LENGTH:
             text_sents = text_sents[:MAX_INPUT_SEQ_LENGTH]
+        elif len(text_sents) == 1:
```

Comment:
why == 1?

Reply:
We do have news that contains only a single sentence. If that's the case, the script cannot calculate sumbasic bigram score (sum_bigram_score = cum_prob_bigram / (len(text_sents) - 1)). So I just ignored any news that has only 1 sentence - these are short enough.